asm/macros/battle_script.inc

```diff
@@ -1713,6 +1713,12 @@
 	various \battler, VARIOUS_JUMP_IF_SHIELDS_DOWN_PROTECTED
 	.4byte \ptr
 	.endm
+
+	.macro trypoisontype battlerability:req, battlertypes:req, ptr:req
```

Comment:
Could battlerability battlertypes be renamed to simply attacker and target?

Reply:
Done.